pkg/mount/mountutil.go

```diff
@@ -79,7 +80,17 @@ func (m DiskMountUtil) getDeviceMountAttr(fn getMountData) (DeviceMountAttr, err
 	if err != nil {
 		return mountAttr, err
 	}
-	defer file.Close()
+
+	defer func() {
```

Comment:
The defer block will be executed only when the function is returned. So why is the `err` assignment done?

Reply:
Fixed